build.ps1

```diff
@@ -21,6 +21,10 @@ Parameters:
     -stresstests: Runs Stress tests.
     -publish: (Internal use, requires nuget toolset) Publishes the nuget packages.
     -verbosity: Sets the verbosity level of the command. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+	-framework: Select which framework to run tests on. Allowed values examples include, but are not limited to, "netcoreapp2.1", "net47", "net451"
```

Comment:
fix spacing to be the same as the others above

Reply:
sure thing